girder/models/upload.py

```diff
@@ -183,16 +183,20 @@ def finalizeUpload(self, upload, assetstore=None):
         if 'fileId' in upload:  # Updating an existing file's contents
             file = File().load(upload['fileId'], force=True)
 
+            if upload.get('attachParent'):
+                item = Item().load(upload['parentId'], force=True)
+            else:
+                item = Item().load(file['itemId'], force=True)
+
+            File().propagateSizeChange(item, upload['size'] - file['size'])
```

Comment:
Do attached files count toward item sizes?  I didn't think so (but don't know for sure).  In which case, this should only be in the non-attached half of the if/else statement.

Reply:
I checked -- attached files don't count toward the parent's size (and therefore are not subject to the user quota either).

Reply:
I was questioning this too, is there a simple way to check if they affect it?